src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
+
+Ideas of "hacking" and "making" have also been popularized as approaches to creatively and collaboratively solve problems in society, business and government.
+
+Where does Inclusive Design fit into these approaches?
+
+## On Terminology
+
+### Hackathon
+
+We use the term "hackathon" expansively in this entry to refer to events that emphasize intensive short-term collaboration between small groups. The groups typically ideate, design and prototype around themes such as <a href="http://arthackathon.co.uk/" target="_blank" class="link-external">art</a> or <a href="http://civictech.ca/2016/04/08/codeacross-toronto-community-collaboration-in-celebration-of-international-open-data-day/" target="_blank" class="link-external">open data</a>, then present their result in some form to to the larger group throughout the event or at the end.
+
+While these events are often focused on producing or proposing new software or hardware as a means of addressing a social problem or creating a new product, this is not stricly necessary; hackathons have been held to produce
+<a href="https://www.policyhack.com.au/" target="_blank" class="link-external">government policy</a>, or ideas to address <a href="http://foodinnovationprogram.org/hacking-hunger/" target="_blank" class="link-external">food security</a>.
+
+In learning settings hackathons might be held as part of course curriculum or as an extracurricular activity for interested students. Academic institutions might seek to partner with community groups for hackathons to encourage students and faculty to form connections outside the institution.
+
+### Hacking
+
+We use the term "hacking" expansively here to refer to ideas of in-depth learning by doing, do-it-yourself and <a href="https://en.wikipedia.org/wiki/Hacker_culture" target="_blank" class="link-external">"playful cleverness"</a> that we can also find present in the term "making". We feel (though others might disagree) that the concepts have enough overlap to discuss them together.
+
+We also consider the hacking approach to be present in one-to-one customizations, adaptations and transformations of technology that allow participation in activities or the use of technology by more diverse groups, including people with disabilities. This approach is found in groups like <a href="http://www.tetrasociety.org/" target="_blank" class="link-external">The Tetra Society of North America</a>, who create customized assistive devices for individuals.
+
+In learning settings, hacking might include classroom projects involving prototyping a product or system, or customizing devices to suit someone's unique needs.
+
+## Inclusively Designing Hackathons and Hacking
+
+It's been said about hackathons that the connections and ideas that get made between people attending are more important than any products that get produced. If hackathons are about building community, issues of inclusion, exclusion and audience diversity are paramount to their design.
```

Comment:
I often feel a bit weird about using the phrase "it's been said..." rather than a more specific citation. Maybe in this case, it would be better to just go ahead and own the (quite interesting and reasonable) assertion that the important outcome of a hackathon is the connections, ideas, starting points and relationships that are formed from the process?